src/main/java/com/gmail/nossr50/skills/salvage/Salvage.java

```diff
@@ -15,7 +22,7 @@
     public static boolean arcaneSalvageDowngrades  = AdvancedConfig.getInstance().getArcaneSalvageEnchantDowngradeEnabled();
     public static boolean arcaneSalvageEnchantLoss = AdvancedConfig.getInstance().getArcaneSalvageEnchantLossEnabled();
 
-    protected static int calculateSalvageableAmount(short currentDurability, short maxDurability, int baseAmount) {
+    static int calculateSalvageableAmount(int currentDurability, short maxDurability, int baseAmount) {
```

Comment:
Why was the type changed?

Reply:
The reason behind it was that Bukkit changed the type of durability from `short` to `int` quite a while ago.
So having it as a short, when it's actually an int seems a bit nonsensical.

Reply:
Do you know when this was changed? mcMMO is currently supporting versions between 1.13.2 -> Latest

Reply:
This happened in the 1.13 Update, since Mojang removed meta/sub ids and everything became its own Material.
Since durability was just a fancy word for meta ids, it just happened to be a side effect of it but yeah, it's been an `int` since 1.13 and part of the `ItemMeta` now.
So no worries, this won't break anything, in fact since the project is built against 1.13 the previous `getDurability()` method which used to return a `short` also shows up with a deprecation warning.